packages/backend/package.json

```diff
@@ -30,7 +30,6 @@
     "@types/tmp": "^0.2.6",
     "@types/uuid": "^9.0.7",
     "cross-fetch": "^4.0.0",
-    "ilp-protocol-ildcp": "^2.2.3",
```

Comment:
We had this in both `dependencies` and `devDependencies`. Not sure why but we shouldn't need it as both so I removed the dev one.